options/linux/include/sys/inotify.h

```diff
@@ -43,7 +44,14 @@ struct inotify_event {
 	unsigned int mask;
 	unsigned int cookie;
 	unsigned int len;
+
+/*
+ * glibc uses a flexible array member here, but we get a warning and they don't:
+ * https://gcc.gnu.org/bugzilla/show_bug.cgi?id=117241
+ */
+#if !defined(__MLIBC_PEDANTIC_HEADER_CHECKER)
 	char name[];
+#endif
```

Comment:
Can't we use some `#pragma GCC diagnostic error` here to suppress this instead of a macro?

Reply:
Done